### PR TITLE
feat(account-lib): add support for ecdsa sigining and verification tss

### PR DESCRIPTION
**Pull request by sachushaji**
this change add support for ecdsa signing and verification for TSS

TICKET: BG-48874

### CODE REVIEW

File: modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts [hunk 1]
```diff
@@ -102,4 +125,240 @@ export default class Ecdsa {
...
+      delete shareParticipant['m'];
+      delete shareToBeSend['alpha'];
+      delete shareToBeSend['mu'];
+    }
+    if (shareToBeSend['k']) {
+      const n = BigInt(shareToBeSend['n']); // Pallier pub from other signer
+      let pk = getPaillierPublicKey(n);
+      const k = BigInt(shareToBeSend['k']);
+
+      const beta0 = Ecdsa.curve.scalarRandom();
```
Comment (sachushaji) [inline modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts:221]: beta0 and nu0, the original range was to from 1 to n (Where n is paillier constant for its public key), But the scalar negation for noble only permits the operation within the secp256k1 curve range, so had to stick with scalar random, @johnoliverdriscoll hope its okay

**Review by olibeyene — DISMISSED** [review]
lgtm!

File: modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts (hunk 1 above)
Comment (johnoliverdriscoll) [reply modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts:223]: I've looked over another implementation and it seems to use a beta value that is within secp256k1's curve order, so I'm sure it's fine this way.

**Review by johnoliverdriscoll — DISMISSED** [review]

**Review by wesgraham — APPROVED** [review]

**Review by bitgoAaron — APPROVED** [review]

File: modules/sdk-core/src/account-lib/mpc/util.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import * as paillierBigint from 'paillier-bigint';
```
[the remark this answers is not in the document]
Comment (sachushaji) [reply modules/sdk-core/src/account-lib/mpc/util.ts:1]: Good Point, I will have it updated

File: modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts (hunk 1 above)
Comment (johnoliverdriscoll) [reply modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts:223]: Ok I had a review of the white paper and I have found that this value (and n0) needs to be within 1 <= beta0 < n. To negate it on the curve you'll simply have to call curve.scalarReduce on it first.

**Pull request merged**